pt/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: news_post
+title: "CVE-2017-0898: Buffer underrun vulnerability in Kernel.sprintf"
+author: "usa"
+translator: "jcserracampos"
+date: 2017-09-14 12:00:00 +0000
+tags: segurança
+lang: pt
+---
+
+Existe uma vulnerabilidade de buffer underrun no método `sprintf` do módulo `Kernel`.
+Essa vulnerabilidade foi assinada com o identificador CVE [CVE-2017-0898](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-0898).
+
+## Detalhes
+
+Se uma _string_ formatada maliciosamente contendo um especificador (`*`) for passada e um valor negativo enorme também for passado para o especificador, poderá causar buffer underrun.
```

Comment:
@jcserracampos aqui vc usou muito bem a formatação de _string_. Acho bacana preserver palavras em outro idioma também em itálico. O que acha?

Neste caso as atualizações seriam simples, o texto na linha `11` ficaria assim:
```
Existe uma vulnerabilidade de _buffer underrun_ no método `sprintf` do módulo `Kernel`.
```

Na linha `16`
```
Se uma _string_ formatada maliciosamente contendo um especificador (`*`) for passada e um valor negativo enorme também for passado para o especificador, poderá causar _buffer underrun_.
```

Na linha `17`
```
Nessa situação, o resultado poderá conter _heap_ de memória ou o interpretador Ruby poderá parar.
```

É isso, estes são meus pontos de vista, no geral PARABÉNS pela excelente tradução e agilidade! 👏

Reply:
Eu fiquei na dúvida sobre esses termos técnicos e não sabia onde levantar a discussão, se seria aqui nas issues ou outro lugar.
Vou começar a utilizar o itálico nesses casos.

E obrigado.

Reply:
👍